Gulpfile.babel.js

```diff
@@ -24,6 +24,10 @@ let modulesPath = path.join(basePrefix, "node_modules");
 let webpackConfig = {
   quiet: false,
   verbose: true,
+  node: {
+    fs: "empty",
+    child_process: "empty"
+  },
```

Comment:
I assume this change was automatically generated, correct?

Reply:
No I had to add it to fix breakage in some node packages.

Reply:
ah, okay